Installation/CHANGES.md

```diff
@@ -106,6 +106,7 @@ extracted from labeled images.
 
 ### [2D Minkowski Sums](https://doc.cgal.org/5.6/Manual/packages.html#PkgMinkowskiSum2)
 -   Fixed a bug that made holes in the Minkowski sum disappear
+-   Fixed hole filtering. The code use to erroneously remove holes from the container of holes of polygons with holes that did not affect the mink. sum. Now it simply adds those that do affect it.
```

Comment:
use -> used
mink. sum -> Minkowski Sum

Reply:
```suggestion
Fixed hole filtering. Previously the code erroneously removed holes from the container of holes of polygons with holes that did not affect the Minkowski sum. Now it simply adds those that do affect it.
```

Reply:
@albert-github Should it be 'affect', or 'effect'?

Reply:
In my email I saw also the question: "Should it be 'affect', or 'effect'?", in my opinion it should be "affect".

Reply:
Efi, you said you would commit that change, but it is not pushed.